### PR TITLE
Build User class and methods and event listeners for nav bar tabs

### LINKED ISSUE #1: Nav  Bar
- Brighter Color - Dark green to match Thyme photo
- Light text color
- Box Shadow
- Includes: Home,  Saved Recipes, Shopping Cart
- Present in all HTML files

### PR DESCRIPTION
**Pull request by zwsaile**
#### What's this PR do?
- Build user class and methods, along with testing file. Sets up event listeners for using all nav bar buttons.

#### Where should the reviewer start?
- User.js and User-test.js

#### How should this be manually tested?
- npm test in the command line

#### Any background context you want to provide?
- Will use this more in depth in Part 2

#### Questions:
- Is user's "pantry" what we need to subtract from the shopping list?

### CODE REVIEW

File: src/classes/RecipeCard.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ getCostOfIngredients(ingredients) {
     })
     return num
   }, 0)
-  return `$${ingredientList/100}`
+  return `$${(ingredientList/100).toFixed(2)}`
```
Comment (sted1994) [inline src/classes/RecipeCard.js:34]: Good idea to keep numbers from repeating

File: test/RecipeCard-test.js [hunk 2]
```diff
@@ -42,15 +41,15 @@ it('should store the tags of the recipe', () => {
 })
 
 it('should determine the names of ingredients needed', () => {
-  expect(recipeCard.getIngredients(ingredientsData)).to.includes('salt')
+  expect(recipeCard.getIngredients(ingredientsData)).to.includes('0.50 tsp salt')
 })
```
Comment (Wperricone) [inline test/RecipeCard-test.js:46]: This is some high quality code!

**Review by Wperricone — APPROVED** [review]

File: src/scripts.js [hunk 3]
```diff
@@ -16,52 +16,110 @@ import './data/users';
...
+const myRecipesTab = document.querySelector('.saved-recipes')
+const shoppingTab = document.querySelector('.shopping-list')
 
 document.addEventListener('keypress', function(event) {
   if(event.key === "Enter"){
     const newRecipeRepository = new RecipeRepository(recipeData);
     newRecipeRepository.getRecipesBySearch(searchInput.value)
-    toggleView(mainPage)
-    toggleView(recipeSelectionPage)
+    hideElement(mainPage)
```
Comment (sted1994) [inline src/scripts.js:43]: Lets make a function that does this

**Review by sted1994 — APPROVED** [review]
Great work! Looks really good

**Pull request merged**